docs/DEVELOPMENT.md

```diff
@@ -28,3 +28,8 @@ dev=1 nix build .#iso-aarch64 --impure
 The `--impure` flag tells nix to allow reading environment variables, and the `dev=1` env var actually tells our build process to use the `dev.json` file as input overrides for all our services.
 
 A couple of our projects that use Golang may need to be explicitly "vendored" to support this workflow. If this is the case, cd to the project directory and run `go mod vendor`. The created `vendor` folder should **not** be committed, and should be added to the `.gitignore` file of the project if it doesn't already exist.
+
+If you get an error opening dev.json, consider clearing your cache with:
```

Comment:
In addition to this, I'd suggest the user also re-run the `git` commands above to re-add the `dev.json` to their git repository before running any further builds.